components/function-runtimes/nodejs10/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM kubeless/nodejs@sha256:aa819e9bee6c22db0b137595ac45765ef1cad64d0dc09502a008ce4446c607ef as source
```

Comment:
The `kubeless/nodejs` image is base on `Debian GNU/Linux 9 (stretch)` and weights `610MB`.
The size of the `/kubeless_rt` directory that we copy is `5.5M`.

Can we somehow improve this?
The same question applies for python image.

Reply:
I'd rather be clear about what our sources are and from which image we've extracted those files

Reply:
[kubeless/runtimes](https://github.com/kubeless/runtimes/tree/master/stable/nodejs) repository contains all the files (including `Dockerfiles`), lets investigate if we can improve this in a followup task;